packages/core/android/src/main/kotlin/com/datadog/reactnative/DdSdkConfigurationExt.kt

```diff
@@ -120,26 +198,23 @@ internal fun ReadableArray.asFirstPartyHosts(): Map<String, Set<TracingHeaderTyp
      */
     val firstPartyHostsWithHeaderTypes = mutableMapOf<String, MutableSet<TracingHeaderType>>()
 
-    val firstPartyHosts = this.toArrayList() as List<ReadableMap>
-    for (it in firstPartyHosts) {
-        val match = it.getString("match")
-        val propagatorTypes = it.getArray("propagatorTypes")?.asTracingHeaderTypes()
-        if (match != null && propagatorTypes != null && propagatorTypes.isNotEmpty()) {
-            val hostMatch = firstPartyHostsWithHeaderTypes[match]
+    for (it in this) {
```

Comment:
`it` is a reserved keyword in Kotlin, so it is better to avoid using it.

```suggestion
    for (host in this) {
```